src/cgame/cg_ents.cpp

```diff
@@ -1379,3 +1379,21 @@ void CG_AddPacketEntities()
 		}
 	}
 }
+
+// see also G_Team in src/sgame/sg_ents.cpp
+team_t CG_Team(const entityState_t &es)
+{
+	switch (es.eType)
+	{
+		case entityType_t::ET_PLAYER:
+		case entityType_t::ET_CORPSE:
```

Comment:
What's the sense of a corpse being on a team? They don't even have any effect on gameplay.

Reply:
I don't think it's used in cgame, but it allows for more consistency with sgame, that does this too.